README.md

```diff
@@ -28,3 +28,22 @@ Finally add a local administrative user, for example:
 ```
 wp user create admin admin@local.host --role=administrator --user_pass=changeme
 ```
+
+## Pull Requests
+
+* If you only have one PR open at a time then you can use the master branch on your fork.
+* If you want to have multiple PRs open at once (which sometimes happens without being planned for) then each one will need its own branch.
+* Best practice is to use a separate branch for each change but if you are only doing one change at a time it doesn't matter much.
+* Make sure to reset to `upstream/master` or the "latest official" code before starting a branch or PR.
```

Comment:
```suggestion
## Pull Requests

We accept pull requests for this website via GitHub. Many of the guidelines at https://github.com/ClassicPress/ClassicPress/blob/develop/.github/CONTRIBUTING.md also apply here, for example:

 - Please be sure to test your PR on a local development installation as described above, or at least using the browser development tools.
 - Please include screenshots of the site before and after your changes as well as an explanation of what has been changed and why.
 - Some parts of the above guidelines are specific to ClassicPress core and do not apply here, like the instructions for backporting changes from WP and running the automated tests.

Please always use a **separate branch** for each change, to avoid issues with your `master` branch not being reset or updated correctly in between different pull requests. If you're not sure how to do this, you can use the GitHub web interface to edit files, open a new branch, and propose your changes as a pull request.
```